pubspec.yaml

```diff
@@ -1,21 +1,21 @@
 name: animated_text_kit
 description: A flutter package project which contains a collection of cool and beautiful text animations.
-version: 3.1.2
+version: 4.0.0
 homepage: https://github.com/aagarwal1012/Animated-Text-Kit/
 maintainer: Ayush Agarwal (@aagarwal1012)
 
 dependencies:
   flutter:
     sdk: flutter
-  characters: ^1.0.0
+  characters: ^1.1.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  pedantic: ^1.9.2
+  pedantic: ^1.10.0
 
 flutter:
 
 environment:
-  sdk: ">=2.10.0 <3.0.0"
+  sdk: '>=2.12.0-259.12.beta <3.0.0'
```

Comment:
I'd like to see double-quotes used because that matches the _Format on Save_ default for `yaml`.

I wonder if this should be so specific and instead just say:
```yaml
sdk: ">=2.12.0 <3.0.0"
```